doc/source/about_geoutils.md

```diff
@@ -0,0 +1,68 @@
+(about-geoutils)=
+
+# About GeoUtils
+
+## What is GeoUtils?
+
+GeoUtils<sup>1</sup> is a **[Python](https://www.python.org/) package for the handling and analysis of georeferenced data**, developed with the objective of
+making such analysis accessible, efficient and reliable.
+
+```{margin}
+<sup>1</sup>With name standing for *Geospatial Utilities*.
+```
+
+In a few words, GeoUtils can be described as a **convenience package for end-users focusing on geospatial analysis**. It allows to write shorter
+code through consistent higher-level operations, implicit object behaviour and interfacing. In addition, GeoUtils adds several analysis-oriented
+functions that require many steps to perform with other packages, and which are robustly tested.
+
+GeoUtils is designed for all Earth and planetary observation science. However, it is generally **most useful for surface applications that rely on
+moderate- to high-resolution data** (requiring reprojection, re-gridding, point interpolation, and other types of fine-grid analysis).
+
+## Why use GeoUtils?
+
+GeoUtils is built on top of [Rasterio](https://rasterio.readthedocs.io/en/latest/), [GeoPandas](https://geopandas.org/en/stable/docs.html)
+and [PyProj](https://pyproj4.github.io/pyproj/stable/index.html) for georeferenced operations, and relies on [NumPy](https://numpy.org/doc/stable/),
+[SciPy](https://docs.scipy.org/doc/scipy/) and [Xarray](https://docs.xarray.dev/en/stable/) for scientific computing to provide:
+- A **common and consistent framework** for efficient raster and vector handling,
+- A structure following the **principal of least knowledge**<sup>2</sup> to foster accessibility,
+- A **pythonic arithmetic** and **NumPy interfacing** for robust numerical computing.
+
+```{margin}
+<sup>2</sup>Or the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter) for software development.
+```
+
+In particular, GeoUtils:
+- Rarely requires more than **single-line operations** thanks to its object-based structure,
+- Strives to rely on **lazy operations** under-the-hood to avoid unnecessary data loading,
+- Allows for **match-reference operations** to facilitate geospatial handling,
```

Comment:
I'm not sure this is clear to everyone at this stage. Maybe point to a later explanation?

Reply:
Was done at the bottom of the paragraph